docs/en/ufs/HDFS.md

```diff
@@ -257,9 +257,9 @@ This will provide a jar that should be moved to the `lib/` directory in the Allu
 
 ```console
 cd shaded/hadoop/
-mvn -T 4C -am clean install -Dmaven.javadoc.skip=true -DskipTests -Dlicense.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Pmesos -Pufs-hadoop-1 -Dufs.hadoop.version=1.2.0
```

Comment:
instead of talking about how to build HDFS with different versions both here,
just referencing  https://github.com/Alluxio/alluxio/blob/master/docs/en/contributor/Building-Alluxio-From-Source.md ? So in the future, we only need to update one place if the way to build changes